msteams-platform/m365-apps/prerequisites.md

```diff
@@ -71,14 +71,22 @@ For more information on Microsoft 365 release options, see [set up the Standard
 
 ### Desktop
 
-You can preview Teams apps running in Outlook on Windows desktop by using a recent *Current Channel (Preview)* build. Check if you have to [change the Microsoft 365 Apps update channel](/deployoffice/change-update-channels?WT.mc_id=M365-MVP-5002016) for your test tenant to install a Microsoft 365 Current Channel (Preview) build.
+Based on the scenario you want to test, you can preview Teams apps running in Outlook on Windows desktop by using a [Microsoft 365 Apps update channel](/deployoffice/change-update-channels?WT.mc_id=M365-MVP-5002016) for your tenant.
 
 To install Microsoft 365 applications in your test environment:
 
 1. Sign in to your test environment with your test tenant credentials.
 1. Download and run the [Office Deployment Tool](https://www.microsoft.com/download/details.aspx?id=49117).
 1. Select a local folder to store the extracted files.
-1. Go to the local folder and open *configuration-Office365-x86.xml* (or **x64.xml*, depending on your environment) in a text editor and ensure the *Channel* value is `CurrentPreview`.
+1. Go to the local folder and open *configuration-Office365-x86.xml* (or **x64.xml*, depending on your environment) in a text editor. Ensure the [*Channel*](/DeployOffice/office-deployment-tool-configuration-options#channel-attribute-part-of-updates-element) value is set according to the scenario you're testing:
+
+    | **Outlook scenario** | **Update channel** | 
+    |---------------|--------------|
+    |Teams personal tabs in Outlook | `CurrentPreview`|
+    |Teams search-based message extensions in Outlook | `CurrentPreview`|
```

Comment:
Since this has been in review, Outlook unified store is now 100% GA. Let's update these 2 rows (tabs,MEs) to say `CurrentChannel` to reflect general availability!